### ISSUE TITLE
`make DESTDIR="/path/to/dir" install` is broken

### ISSUE THREAD

**Issue by orestisfl** [rockowitz/ddcutil#329]
After https://github.com/rockowitz/ddcutil/commit/994f4aa8fe4f514d42a4022ac70e7358c9dc7b79, it seems that running `make install` with a custom `DESTDIR` is not working:

```
Making install in src
make[1]: Entering directory '/tmp/xx/ddcutil/src'
Making install in util
make[2]: Entering directory '/tmp/xx/ddcutil/src/util'
make[3]: Entering directory '/tmp/xx/ddcutil/src/util'
make[3]: Nothing to be done for 'install-exec-am'.
make[3]: Nothing to be done for 'install-data-am'.
make[3]: Leaving directory '/tmp/xx/ddcutil/src/util'
make[2]: Leaving directory '/tmp/xx/ddcutil/src/util'
Making install in usb_util
make[2]: Entering directory '/tmp/xx/ddcutil/src/usb_util'
make[3]: Entering directory '/tmp/xx/ddcutil/src/usb_util'
make[3]: Nothing to be done for 'install-exec-am'.
make[3]: Nothing to be done for 'install-data-am'.
make[3]: Leaving directory '/tmp/xx/ddcutil/src/usb_util'
make[2]: Leaving directory '/tmp/xx/ddcutil/src/usb_util'
Making install in base
make[2]: Entering directory '/tmp/xx/ddcutil/src/base'
make[3]: Entering directory '/tmp/xx/ddcutil/src/base'
make[3]: Nothing to be done for 'install-exec-am'.
make[3]: Nothing to be done for 'install-data-am'.
make[3]: Leaving directory '/tmp/xx/ddcutil/src/base'
make[2]: Leaving directory '/tmp/xx/ddcutil/src/base'
Making install in vcp
make[2]: Entering directory '/tmp/xx/ddcutil/src/vcp'
make[3]: Entering directory '/tmp/xx/ddcutil/src/vcp'
make[3]: Nothing to be done for 'install-exec-am'.
make[3]: Nothing to be done for 'install-data-am'.
make[3]: Leaving directory '/tmp/xx/ddcutil/src/vcp'
make[2]: Leaving directory '/tmp/xx/ddcutil/src/vcp'
Making install in i2c
make[2]: Entering directory '/tmp/xx/ddcutil/src/i2c'
make[3]: Entering directory '/tmp/xx/ddcutil/src/i2c'
make[3]: Nothing to be done for 'install-exec-am'.
make[3]: Nothing to be done for 'install-data-am'.
make[3]: Leaving directory '/tmp/xx/ddcutil/src/i2c'
make[2]: Leaving directory '/tmp/xx/ddcutil/src/i2c'
Making install in usb
make[2]: Entering directory '/tmp/xx/ddcutil/src/usb'
make[3]: Entering directory '/tmp/xx/ddcutil/src/usb'
make[3]: Nothing to be done for 'install-exec-am'.
make[3]: Nothing to be done for 'install-data-am'.
make[3]: Leaving directory '/tmp/xx/ddcutil/src/usb'
make[2]: Leaving directory '/tmp/xx/ddcutil/src/usb'
Making install in dynvcp
make[2]: Entering directory '/tmp/xx/ddcutil/src/dynvcp'
make[3]: Entering directory '/tmp/xx/ddcutil/src/dynvcp'
make[3]: Nothing to be done for 'install-exec-am'.
make[3]: Nothing to be done for 'install-data-am'.
make[3]: Leaving directory '/tmp/xx/ddcutil/src/dynvcp'
make[2]: Leaving directory '/tmp/xx/ddcutil/src/dynvcp'
Making install in ddc
make[2]: Entering directory '/tmp/xx/ddcutil/src/ddc'
make[3]: Entering directory '/tmp/xx/ddcutil/src/ddc'
make[3]: Nothing to be done for 'install-exec-am'.
make[3]: Nothing to be done for 'install-data-am'.
make[3]: Leaving directory '/tmp/xx/ddcutil/src/ddc'
make[2]: Leaving directory '/tmp/xx/ddcutil/src/ddc'
Making install in libmain
make[2]: Entering directory '/tmp/xx/ddcutil/src/libmain'
make[3]: Entering directory '/tmp/xx/ddcutil/src/libmain'
make[3]: Nothing to be done for 'install-exec-am'.
make[3]: Nothing to be done for 'install-data-am'.
make[3]: Leaving directory '/tmp/xx/ddcutil/src/libmain'
make[2]: Leaving directory '/tmp/xx/ddcutil/src/libmain'
Making install in app_sysenv
make[2]: Entering directory '/tmp/xx/ddcutil/src/app_sysenv'
make[3]: Entering directory '/tmp/xx/ddcutil/src/app_sysenv'
make[3]: Nothing to be done for 'install-exec-am'.
make[3]: Nothing to be done for 'install-data-am'.
make[3]: Leaving directory '/tmp/xx/ddcutil/src/app_sysenv'
make[2]: Leaving directory '/tmp/xx/ddcutil/src/app_sysenv'
Making install in app_ddcutil
make[2]: Entering directory '/tmp/xx/ddcutil/src/app_ddcutil'
make[3]: Entering directory '/tmp/xx/ddcutil/src/app_ddcutil'
make[3]: Nothing to be done for 'install-exec-am'.
make[3]: Nothing to be done for 'install-data-am'.
make[3]: Leaving directory '/tmp/xx/ddcutil/src/app_ddcutil'
make[2]: Leaving directory '/tmp/xx/ddcutil/src/app_ddcutil'
Making install in cmdline
make[2]: Entering directory '/tmp/xx/ddcutil/src/cmdline'
make[3]: Entering directory '/tmp/xx/ddcutil/src/cmdline'
make[3]: Nothing to be done for 'install-exec-am'.
make[3]: Nothing to be done for 'install-data-am'.
make[3]: Leaving directory '/tmp/xx/ddcutil/src/cmdline'
make[2]: Leaving directory '/tmp/xx/ddcutil/src/cmdline'
Making install in .
make[2]: Entering directory '/tmp/xx/ddcutil/src'
make[3]: Entering directory '/tmp/xx/ddcutil/src'
 /usr/bin/mkdir -p '/tmp/xx/dest//usr/lib'
 /bin/sh ../libtool   --mode=install /usr/bin/install -c   libddcutil.la '/tmp/xx/dest//usr/lib'
libtool: install: /usr/bin/install -c .libs/libddcutil.so.5.0.0 /tmp/xx/dest//usr/lib/libddcutil.so.5.0.0
libtool: install: (cd /tmp/xx/dest//usr/lib && { ln -s -f libddcutil.so.5.0.0 libddcutil.so.5 || { rm -f libddcutil.so.5 && ln -s libddcutil.so.5.0.0 libddcutil.so.5; }; })
libtool: install: (cd /tmp/xx/dest//usr/lib && { ln -s -f libddcutil.so.5.0.0 libddcutil.so || { rm -f libddcutil.so && ln -s libddcutil.so.5.0.0 libddcutil.so; }; })
libtool: install: /usr/bin/install -c .libs/libddcutil.lai /tmp/xx/dest//usr/lib/libddcutil.la
libtool: warning: remember to run 'libtool --finish /usr/lib'
 /usr/bin/mkdir -p '/tmp/xx/dest//usr/bin'
  /bin/sh ../libtool   --mode=install /usr/bin/install -c ddcutil '/tmp/xx/dest//usr/bin'
libtool: install: /usr/bin/install -c ddcutil /tmp/xx/dest//usr/bin/ddcutil
make  install-exec-hook
make[4]: Entering directory '/tmp/xx/ddcutil/src'
(src/Makefile) Executing install-exec-hook ...
rm -f /tmp/xx/dest//usr/lib/libddcutil.la  
make[4]: Leaving directory '/tmp/xx/ddcutil/src'
 /usr/bin/mkdir -p '/tmp/xx/dest//usr/include'
 /usr/bin/install -c -m 644 public/ddcutil_macros.h public/ddcutil_status_codes.h public/ddcutil_types.h public/ddcutil_c_api.h '/tmp/xx/dest//usr/include'
make[3]: Leaving directory '/tmp/xx/ddcutil/src'
make[2]: Leaving directory '/tmp/xx/ddcutil/src'
Making install in sample_clients
make[2]: Entering directory '/tmp/xx/ddcutil/src/sample_clients'
make[3]: Entering directory '/tmp/xx/ddcutil/src/sample_clients'
make[3]: Nothing to be done for 'install-exec-am'.
make[3]: Nothing to be done for 'install-data-am'.
make[3]: Leaving directory '/tmp/xx/ddcutil/src/sample_clients'
make[2]: Leaving directory '/tmp/xx/ddcutil/src/sample_clients'
make[1]: Leaving directory '/tmp/xx/ddcutil/src'
Making install in man
make[1]: Entering directory '/tmp/xx/ddcutil/man'
make[2]: Entering directory '/tmp/xx/ddcutil/man'
make[2]: Nothing to be done for 'install-exec-am'.
 /usr/bin/mkdir -p '/tmp/xx/dest//usr/share/man/man1'
 /usr/bin/install -c -m 644 ddcutil.1 '/tmp/xx/dest//usr/share/man/man1'
make[2]: Leaving directory '/tmp/xx/ddcutil/man'
make[1]: Leaving directory '/tmp/xx/ddcutil/man'
Making install in data
make[1]: Entering directory '/tmp/xx/ddcutil/data'
(data/Makefile) ==>  Executing rule: all-local
make[2]: Entering directory '/tmp/xx/ddcutil/data'
make[2]: Nothing to be done for 'install-exec-am'.
 /usr/bin/mkdir -p '/tmp/xx/dest//usr/lib/cmake/ddcutil'
 /usr/bin/install -c -m 644 cmake/ddcutil/FindDDCUtil.cmake '/tmp/xx/dest//usr/lib/cmake/ddcutil'
(data/Makefile) ==>  Executing rule: install-data-local
prefix:         /usr
includedir      /usr/include
docdir          /usr/share/doc/ddcutil
libdir          /usr/lib
rulesdir        /usr/lib/udev/rules.d
packagedatadir: 
datadir:        /usr/share
ddcutildir:     /usr/share/ddcutil/data
srcdir:         .
bindir:         /usr/bin
cmakedir:       /usr/lib/cmake/ddcutil
DESTDIR:        /tmp/xx/dest/
rulesfiles:     usr/lib/udev/rules.d/60-ddcutil-i2c.rules usr/lib/udev/rules.d/60-ddcutil-usb.rules usr/lib/udev/rules.d/60-libddcutil-i2c.rules
resfiles:       etc/modprobe.d/nvidia-i2c.conf etc/udev/rules.d/60-ddcutil-i2c.rules etc/udev/rules.d/60-ddcutil-usb.rules etc/X11/xorg.conf.d/90-nvidia-i2c.conf
udevdir:        
 /usr/bin/mkdir -p '/tmp/xx/dest//usr/share/ddcutil/data'
 /usr/bin/install -c -m 644 etc/modprobe.d/nvidia-i2c.conf etc/udev/rules.d/60-ddcutil-i2c.rules etc/udev/rules.d/60-ddcutil-usb.rules etc/X11/xorg.conf.d/90-nvidia-i2c.conf '/tmp/xx/dest//usr/share/ddcutil/data'
 /usr/bin/mkdir -p '/tmp/xx/dest//usr/lib/modules-load.d'
 /usr/bin/install -c -m 644 usr/lib/modules-load.d/ddcutil.conf '/tmp/xx/dest//usr/lib/modules-load.d'
 /usr/bin/mkdir -p '/tmp/xx/dest//usr/lib/udev/rules.d'
 /usr/bin/install -c -m 644 usr/lib/udev/rules.d/60-ddcutil-i2c.rules usr/lib/udev/rules.d/60-ddcutil-usb.rules usr/lib/udev/rules.d/60-libddcutil-i2c.rules '/tmp/xx/dest//usr/lib/udev/rules.d'
make  install-data-hook
make[3]: Entering directory '/tmp/xx/ddcutil/data'
(data/install-data-hook)===> Executing rule: install-data-hook
sed  -i "s|/usr|/usr|" /usr/lib/udev/rules.d/60-ddcutil-usb.rules 
sed: can't read /usr/lib/udev/rules.d/60-ddcutil-usb.rules: No such file or directory
make[3]: *** [Makefile:701: install-data-hook] Error 2
make[3]: Leaving directory '/tmp/xx/ddcutil/data'
make[2]: *** [Makefile:604: install-data-am] Error 2
make[2]: Leaving directory '/tmp/xx/ddcutil/data'
make[1]: *** [Makefile:556: install-am] Error 2
make[1]: Leaving directory '/tmp/xx/ddcutil/data'
make: *** [Makefile:543: install-recursive] Error 1
```

Full command used:
```
git clean -fdx . && NOCONFIGURE=1 ./autogen.sh && ./configure --prefix=/usr && make -j && make DESTDIR='/tmp/xx/dest/' install
```

Initially reported at https://aur.archlinux.org/packages/ddcutil-dev-git

**Comment by rockowitz** [comment]
Fixed by 32cd on branch 2.0.0-dev

**Comment by orestisfl** [comment]
Thank you, confirmed
